flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/SortMergeResultPartitionReadScheduler.java

```diff
@@ -85,8 +85,8 @@ class SortMergeResultPartitionReadScheduler implements Runnable, BufferRecycler
     /** Executor to run the shuffle data reading task. */
     private final Executor ioExecutor;
 
-    /** Maximum number of buffers can be allocated by this partition reader. */
-    private final int maxRequestedBuffers;
+    /** Number of subpartitions in the target result partition. */
+    private final int numSubpartitions;
```

Comment:
This filed seems not used so can be removed.

Reply:
OK, fixed it.